client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
 	const oldPackage = newPackages[ packageId ];
 
+	const oldBox = state.form.packages.all[ oldPackage.box_id ];
+	const newBox = state.form.packages.all[ boxTypeId ];
+	const weight =
+		( newBox != null ? newBox.box_weight : 0 ) - ( oldBox != null ? oldBox.box_weight : 0 ) +
```

Comment:
Here we're checking to see if `newBox` is not `undefined` right? If so, we can rely on the falsy if check like so:

`( newBox ? newBox.box_weight : 0 )`